app/src/layouts/common/appSidebar/helpAndService/modal/requestSupportModal.jsx

```diff
@@ -75,25 +78,40 @@ const RequestSupport = ({ handleSubmit, initialize, invalid }) => {
       nextAction();
     };
   };
+  const consentHandler = (e) => {
+    setIsConsentChecked(e.target.checked);
+  };
+
+  const privocyPolicyRef = {
+    a: (
+      <LinkItem
+        link={referenceDictionary.rpEpamPolicy}
+        content={formatMessage(messages.privacyPolicy)}
+        icon={OpenIcon}
+        className={cx('inline-ref')}
+      />
+    ),
+  };
 
   return (
     <ModalLayout
-      title={formatMessage(messages.requestSupport)}
+      title={formatMessage(messages.requestService)}
       okButton={{
-        text: formatMessage(COMMON_LOCALE_KEYS.SEND),
-        onClick: (closeModal) => {
+        text: formatMessage(messages.sendRequest),
+        onClick: () => {
           handleSubmit(() => {
-            onSubmit(closeModal);
+            onSubmit(() => dispatch(hideModalAction()));
           })();
         },
-        disabled: invalid,
+        disabled: invalid || !isConsentChecked,
         attributes: { type: 'submit', form: REQUEST_FORM_ID },
         eventInfo: HELP_AND_SUPPORT_EVENTS.CLICK_SEND_REQUEST_SUPPORT_BUTTON,
       }}
       cancelButton={{ text: formatMessage(COMMON_LOCALE_KEYS.CANCEL) }}
+      onClose={() => dispatch(hideModalAction())}
```

Comment:
Looks the same as
![image](https://github.com/reportportal/service-ui/assets/31861805/24e6a58d-c5b7-430b-bcca-3c509a9dce75)
Please move to a function.